charts/dave/charts/geodata-eai/templates/secret.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "daveGeodataEai.fullname" . }}-secret
+  labels: 
+    {{- include "daveGeodataEai.labels" . | nindent 4 }}
+stringData:
+  {{- with .Values.credentials }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)